apps/src/feedback.js

```diff
@@ -203,7 +203,8 @@ FeedbackUtils.prototype.displayFeedback = function (options, requiredBlocks,
       this.calculateStageProgress(
         actualBlocks <= idealBlocks,
         hintsUsed,
-        options.response.level_id) :
+        options.response.level_id,
+        Number.isFinite(idealBlocks)) :
```

Comment:
annoyingly, `Number.isFinite` is unsupported by IE; you should be able to just use the global `isFinite`, though!